### DISCUSSION TITLE
Score file location

### DISCUSSION THREAD

**Discussion by martinschonger** [empriselab/RCareWorld#195] [Help - Using] [answered]
Hi, it seems that for the bathing task `spongeScore.json` is stored in `~/.config/unity3d/RCareWorld/DressingPlayer/` (instead of `~/.config/unity3d/RCareWorld/BathingPlayer/`). Is that a known issue or am I doing something wrong? Thanks

**Comment by YoruCathy** [comment, marked as answer]
Hi @martinschonger, we are putting both under the dressing path for an easier pipeline in eval ai. Updated the readme instruction to reflect this in #196
